lib/fs/spifs/rules.mk

```diff
@@ -0,0 +1,8 @@
+LOCAL_DIR := $(GET_LOCAL_DIR)
+
+MODULE := $(LOCAL_DIR)
+
```

Comment:
set a MODULE_DEPS on lib/fs and lib/cksum, since both of those apis are used here